js/components/buttons/integrate.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+<button type="button" class="btn btn-primary btn-integrate" :title="title" v-tooltip
+    v-popover popover-large :popover-title="_('Share')">
+    <span class="fa fa-code"></span>
+    <div data-popover-content>
+        <div class="integrate-popover">
+            <p>{{ help }}</p>
+            <div class="integrate-popover-wrapper">
+                <textarea readonly rows="4" v-el:textarea>{{ snippet }}</textarea>
+                <a class="btn btn-link" @click="click"
+                    :title="tooltip" v-tooltip tooltip-placement="bottom">
+                    <span class="fa fa-3x fa-clipboard"></span>
+                </a>
+            </div>
+            <p v-if="documentationUrl">
+                <span class="fa fa-question-circle"><a :href="documentationUrl">{{documentation}}</a></span>
+            </p>
+        </div>
+    </div>
+</button>
+</template>
+
+<script>
+import pubsub from 'pubsub';
+
+export default {
+    props: {
+        title: {type: String, required: true},
+        objectType: {type: String, required: true},
+        objectId: {type: String, required: true},
+        widgetUrl: {type: String, required: true},
+        documentationUrl: String
+    },
+    data() {
+        return {
+            help: this._('Copy-paste this code within your own HTML at the place you want the current dataset to appear:'),
+            tooltip: this._('Click the button to copy the whole code within your clipboard'),
+            documentation: this._('Read the documentation to insert more than one dataset'),
+        };
+    },
+    computed: {
+        snippet() {
+            const tag = 'script'; // Due to vue-loader failing on closing script tag, we interpolate it
```

Comment:
I tried difderent solutions, this one was the easiest and I'm quite proud of if (because it drove me crazy until I understood the issue)